jobListings/createJobListing.js

```diff
@@ -0,0 +1,20 @@
+const jobListings = require("./jobListings-model");
+
+function createJobListing(req, res) {
+  const jobListing = req.body;
+
+  const newJobListing = { ...jobListing };
```

Comment:
This seems to be an unnecessary line of code. It only becomes useful if you're attaching something to jobListing to post.